### DISCUSSION TITLE
Ordered List

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#230] [Component suggestions]
<img width="480" alt="visuele weergave van het ordered list component" src="https://github.com/nl-design-system/backlog/assets/545207/a38bfbb4-e1b1-4f03-849b-aeb37662cc1d">

We willen graag meer leren over het Ordered List component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1710341400011519)

---

## Naam
Ordered List

## Doel
Lijst waarvan de volgorde van items betekenis heeft.

## Gerelateerde componenten
[Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289), [Unordered List](https://github.com/orgs/nl-design-system/discussions/229)

**Comment by jeffreylauwers** [comment]
## Amsterdam

- [Storybook](https://amsterdam.github.io/design-system/?path=/docs/components-text-ordered-list--docs)
- [GitHub](https://github.com/Amsterdam/design-system/tree/develop/packages/css/src/components/ordered-list)
- [Figma](https://www.figma.com/file/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?type=design&node-id=5003-19229&mode=design&t=fR3GEi2CGJzOMP9C-4)

---

### Screenshots / Varianten

**Storybook**

Default
The list uses ascending numbers as markers, providing enough space for numbers up to 99. Extra white space between items enhances the distinction, mainly when they consist of multiple lines of text.
![Scherm­afbeelding 2024-03-14 om 13 23 17](https://github.com/nl-design-system/backlog/assets/545207/c8a3d748-d51c-4e1d-a7f8-cad3a9a2577b)

Two Levels
A list may have one nested level. In this case, lowercase letters are used as markers.
![Scherm­afbeelding 2024-03-14 om 13 23 24](https://github.com/nl-design-system/backlog/assets/545207/b89839e1-fc68-4120-817c-56bd40160931)

Starting Number
When necessary, the list can start with a higher number. Use the HTML attribute `start`.
![Scherm­afbeelding 2024-03-14 om 13 23 29](https://github.com/nl-design-system/backlog/assets/545207/8a86378c-90dd-4ceb-a5ab-7f5c85874426)

Reverse Order
The numbers can also be in reverse order. Use the HTML attribute `reversed`.
![Scherm­afbeelding 2024-03-14 om 13 23 33](https://github.com/nl-design-system/backlog/assets/545207/7ca0d32d-a91e-4e70-b357-e689da2ce9d2)

Inverse Colour
Set the inverseColor prop if the List sits on a dark background. This ensures the colour of the text provides enough contrast. When nesting lists, set the prop on all lists.
![Scherm­afbeelding 2024-03-14 om 13 23 47](https://github.com/nl-design-system/backlog/assets/545207/9a458740-a2e0-45e6-9162-328f89aace09)

Without Markers
An overview of articles or search results can be presented as an ordered can be seen as an unordered list. In such cases, apply the list semantics while hiding markers. This helps screen readers understand and navigate the list.

Note: each item typically contains a container with titles and other text components in this variant. Therefore, it does not define typography for the items and the `inverseColor` prop has no effect.

For example, here are three news articles:
![Scherm­afbeelding 2024-03-14 om 13 23 43](https://github.com/nl-design-system/backlog/assets/545207/ea2b5feb-8cac-4d72-a67b-3d5402d98d87)


**Figma**
![Scherm­afbeelding 2024-03-14 om 11 05 31](https://github.com/nl-design-system/backlog/assets/545207/454c7e7b-6161-4b2a-9c61-f52e5e94c3ce)

---

### Notities
- Storybook component biedt de mogelijkheid om de kleur om te keren bij een donkere achtergrond.
- In Figma wordt het component gecombineerd met de ordered list.
- In Figma de optie om een groen check of rood cross icoon te tonen in plaats van een bullet.
- Documentatie beschikbaar (Storybook).

**Comment by jeffreylauwers** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-data-display-ordered-list--docs)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/OrderedList)
- [Figma - Huidig](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?type=design&node-id=8105-22880&mode=design&t=TDQARyHiaCmjMM77-4)
- [Figma - WIP](https://www.figma.com/file/x4RkF6BIdrNZbh7D53NTzB/%F0%9F%92%A0-NLDS---Den-Haag---Bibliotheek?type=design&node-id=823-595&mode=design&t=X99XUbbTmzlT2yZH-4)

---

### Screenshots / Varianten

**Storybook**

Ordered List with nesting
![Scherm­afbeelding 2024-03-14 om 13 28 26](https://github.com/nl-design-system/backlog/assets/545207/ebb3d6c9-54b5-4c2d-84bb-a4aea199124d)

Ordered list + Paragraph
![Scherm­afbeelding 2024-03-14 om 13 28 31](https://github.com/nl-design-system/backlog/assets/545207/02936505-298a-4918-ab5d-c56a7594af54)

Figma - Huidig
![Scherm­afbeelding 2024-03-14 om 11 17 18](https://github.com/nl-design-system/backlog/assets/545207/fed479cc-85f3-4a39-a37f-c51f477745ca)

Figma - WIP
![Scherm­afbeelding 2024-03-14 om 13 31 52](https://github.com/nl-design-system/backlog/assets/545207/a1fc6bc9-86d1-4f90-b8fd-df66bd39a581)


---

### Notities
- Documentatie beschikbaar.

**Comment by jeffreylauwers** [comment]
## DSO

[Storybook](https://storybook.dso-toolkit.nl/94.0.0/?path=/story/html-css-list--ordered&viewMode=story)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-03-14 om 13 33 10](https://github.com/nl-design-system/backlog/assets/545207/aecf262b-ae1c-4b0c-b54f-41a2f2a1a8ed)

Group
![Scherm­afbeelding 2024-03-14 om 11 30 08](https://github.com/nl-design-system/backlog/assets/545207/6d38bc00-6583-4e34-9f93-5a9de3cc7dff)

Columns
![Scherm­afbeelding 2024-03-14 om 11 30 16](https://github.com/nl-design-system/backlog/assets/545207/51ebf3c1-20cf-4754-9d3c-2228423b7477)

Image list
![Scherm­afbeelding 2024-03-14 om 11 30 23](https://github.com/nl-design-system/backlog/assets/545207/53fb0397-749c-467e-99ac-5857bf7a60f7)

Unstyled
![Scherm­afbeelding 2024-03-14 om 11 32 16](https://github.com/nl-design-system/backlog/assets/545207/5becd8f0-26d9-447c-8937-58ca53b87087)

---

### Notities
- Ondergebracht bij [List](https://storybook.dso-toolkit.nl/62.8.4/?path=/docs/html-css-list--docs).
- In tegenstelling tot het type '[unordered](https://github.com/orgs/nl-design-system/discussions/229#discussioncomment-8784457)' lijkt de icon optie niet beschikbaar voor 'ordered'.

**Comment by jeffreylauwers** [comment]
## DUO

- [Documentatie website](https://uno.duo.nl/#/componenten/list)

---

### Screenshots / Varianten

**Documentatie website**

Decimaal
Gebruik een decimaal lijst bij een duidelijke volgorde in de lijst. Het is mogelijk om de lijst in een column te tonen.
![Scherm­afbeelding 2024-03-14 om 13 35 45](https://github.com/nl-design-system/backlog/assets/545207/0e91e988-be67-460c-95fd-8717a4afc4db)

Decimaal met achtergrond
Gebruik een decimaal lijst met achtergrond de informatie net iets belangrijker is.
![Scherm­afbeelding 2024-03-14 om 13 35 49](https://github.com/nl-design-system/backlog/assets/545207/39836f25-7d58-4822-9a57-c89177671295)

---

### Notities
- Ondergebracht bij Lists.
- Meerdere varianten voor [Unordered list](https://github.com/orgs/nl-design-system/discussions/229#discussioncomment-8784555).

**Comment by jeffreylauwers** [comment]
## Open Gemeenten

- [Documentatie website - Stappenplan](https://componenten.lenteveld.nl/stappenplan)
- [Patternlab](https://patternlab.lenteveld.nl/index.html)

---

### Screenshots / Varianten

**Documentatie website**

![Scherm­afbeelding 2024-03-14 om 13 42 26](https://github.com/nl-design-system/backlog/assets/545207/276d7c56-9eb7-45d1-aa3d-35d6f0bba771)


---

### Notities
- Niet kunnen vinden in Patternlab. Vandaar screenshot documentatie omgeving. Vraag staat uit bij Martijn.

**Comment by jeffreylauwers** [comment]
## Rotterdam

- [Storybook](https://nl-design-system.github.io/rotterdam/?path=/docs/css-utrecht-ordered-list--docs)
- [GitHub](https://github.com/nl-design-system/rotterdam/tree/main/rotterdam-nlds-parent-wicket/docs-wicket/src/main/java/nl/rotterdam/nl_design_system/docs/wicket/ordered_list)
- [Figma](https://www.figma.com/file/ZWSC4gCrOXRUR9UX3aoZ8x/Gemeente-Rotterdam-Design-System?type=design&node-id=490-17441&mode=design&t=BL7EnBsYrSwV4sCp-4)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-03-14 om 13 43 29](https://github.com/nl-design-system/backlog/assets/545207/b752812d-942a-4a1c-bc1f-cd69097e150e)

---

### Notities

**Comment by jeffreylauwers** [comment]
## RVO

- [Documentatie website](https://nl-design-system.github.io/rvo/docs/componenten/ordered-unordered-list/)
- [GitHub](https://github.com/nl-design-system/rvo/tree/master/components/ordered-unordered-list)
- [Storybook](https://nl-design-system.github.io/rvo/?path=/docs/componenten-ordered-unordered-list--documentatie)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-03-14 om 13 45 29](https://github.com/nl-design-system/backlog/assets/545207/d4010a03-a61d-4024-8b37-347d6b599332)

---

### Notities
- Samengevoegd met unordered list.
- In tegenstelling tot [unordered list](https://github.com/orgs/nl-design-system/discussions/229#discussioncomment-8784836) geen andere opties als default.

**Comment by jeffreylauwers** [comment]
## Utrecht

- [Documentatie website](https://nl-design-system.github.io/utrecht/docs/componenten/data%20display/ordered-list)
- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-ordered-list--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/ordered-list)
- [Figma](https://www.figma.com/file/UXIHcIurAD8hyoBWx4hDBV/NLDS---Gemeente-Utrecht---Bibliotheek?type=design&node-id=823-595&mode=design&t=ogYp0zRJUQmFZoCs-4)

---

### Screenshots / Varianten

**Storybook**
Default
![Scherm­afbeelding 2024-03-14 om 13 48 28](https://github.com/nl-design-system/backlog/assets/545207/379cef02-e4ed-405e-9833-4b3e236d85cd)

Right to left
Right-to-left text should have list item markers before the text, on the right. With CSS you can use Arabic script digits for list item numbers, instead of the default latin numbers.
![Scherm­afbeelding 2024-03-14 om 13 48 33](https://github.com/nl-design-system/backlog/assets/545207/0ee4eb19-b971-460f-b6a2-bc001c243671)


**Figma**
![Scherm­afbeelding 2024-03-14 om 13 49 33](https://github.com/nl-design-system/backlog/assets/545207/9bf43473-5847-40b2-bd6e-b0886ff2e483)

---

### Notities

**Reply by Rozerinay** [reply]
Opmerking afkomstig uit een comment in de NL Design System Figma bibliotheek:

Token `utrecht.ordered-list.item.margin-block-start` en `utrecht.ordered-list.item.margin-block-end` zijn in Figma gebruikt als padding, en niet opgenomen in het voorbeeldthema (JSON). Mogelijk vervangen we dit later door `row-gap` voor een duidelijkere scheiding tussen items.

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Ordered list op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=ordered-list)

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Ordered list op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=ordered+list+%23117)

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al ingesteld voor meerdere huisstijlen?

Het kernteam heeft gekeken of Ordered List van de Gemeente Utrecht al geschikt is om richting Hall of Fame te gaan. Hiervoor willen we dat de component succesvol in tenminste 2 verschillende huisstijlen is ingezet.

[We hebben gekeken naar de huisstijl](https://github.com/nl-design-system/candidate/issues/1075#issuecomment-4096656372) van onder andere de Gemeeente Duiven, de Gemeente Haarlemmermeer en de Gemeente Hoorn en we zijn tot de volgende conclusie gekomen:

🎉 Jazeker! Met jullie hulp hebben we genoeg informatie verzameld om een goed instelbaar component als Candidate beschikbaar te gaan maken.

Heb je aanvullingen? Laat het ons onder dit bericht weten!

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al ingezet in meerdere projecten?

Omdat de Ordered List van Gemeente Utrecht geschikt is voor verschillende huisstijlen heeft het kernteam ook gekeken of hij al gebruikt wordt in productie, prototypes en templates.
Deze informatie is nodig om de component een stap verder te brengen in het Candidate proces.

[We hebben hiervoor naar verschillende projecten gekeken](https://github.com/nl-design-system/candidate/issues/1076#issuecomment-4096732175) waarin de component succesvol is ingezet en we zijn tot de volgende conclusie gekomen:

🎉 De component wordt onder andere gebruikt door:

- [Loket Digitaal Utrecht](https://loket.digitaal.utrecht.nl/nl/producten/bezwaar-maken)
- [NL Design System](https://nldesignsystem.nl/wcag/1.3.2/)
- [OpenStad](https://openstad.org/docs/how-to/console-log)
- [Druten Wijchen via OpenStad](https://doemee.drutenwijchen.nl/zon-wind/nieuws/ga-in-gesprek-tijdens-de-dag-van-zon-en-wind-druten)
- [Buurtwarmtenet Watergraafsmeer via OpenStad](https://buurtwarmtenet-watergraafsmeer.nl/veelgestelde-vragen)
- [NORA Online](https://www.noraonline.nl/wiki/Architectuur_beschrijven_binnen_overheidsprojecten)
- [VNG Realisatie FTV](https://vng-realisatie.github.io/ftv/toepassen/implementatie/)
- [Conduction - Common Gateway](https://commongateway.app/)

Heb je aanvullingen? Laat het ons onder dit bericht weten!

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al meegenomen in een WCAG-EM audit?

Om de Ordered List van de Gemeente Utrecht als een toegankelijk Candidate component beschikbaar te maken is het handig als we weten of de Community implementaties al helemaal toegankelijk kunnen worden ingezet.

[We hebben hiervoor gekeken of er een toegankelijkheidsverklaring beschikbaar was](https://github.com/nl-design-system/candidate/issues/1077#issuecomment-4097575841) voor de projecten die in productie worden gebruikt. En:

🎉 Ja! We hebben de uitkomsten van de volgende WCAG-EM audit (1 of meerdere) kunnen vinden:

- [Digitaal Loket Utrecht](https://toegankelijkheidsrapport.frameless.io/rapport/loket.digitaal.utrecht.nl-2024-06)
- [NL Design System](https://nldesignsystem.nl/toegankelijkheidsverklaring/rapport-2024/)
- [Doe mee Druten Wijchen (content)](https://www.wijchen.nl/data/downloadables/1/8/9/0/wcag-22-aa-deelonderzoek-content-doemeedrutenwijchennl-10-202503141111.pdf)
- [OpenStad (techniek)](https://toegankelijkheidsrapport.swink.nl/openstad-hercontrole/audit/)
- [NORA Online](https://toegankelijkheidsrapport.swink.nl/noraonline.nl-hercontrole/audit/)

Heb je aanvullingen? Laat het ons onder dit bericht weten!

**Comment by eslook** [comment]
## Candidate voorbereidingsfase: Issues bekend

We hebben gekeken bij Amsterdam, RVO, Utrecht, Rotterdam, Tilburg, Den Haag en Rijkshuisstijl Community naar openstaande issues en pull requests gerelateerd aan de Ordered List, de input uit de GitHub discussion meegenomen en issues bekend bij de community uitgevraagd. Daaruit zijn de volgende issues bekend gekomen.

We hebben besloten om deze issues wel mee te nemen, om de volgende redenen:

- [Utrecht](https://github.com/nl-design-system/utrecht)
  - [Ordered list has fallback tokens to unordered list, doesn't seem right](https://github.com/nl-design-system/utrecht/issues/2426)
    Dit is relevant omdat componenten niet van elkaars tokens gebruik moeten maken. Hier houden we standaard rekening mee.
  - [Ordered and unordered list have no color tokens](https://github.com/nl-design-system/utrecht/issues/2349)
    Dit is relevant. We hebben meegenomen in het Candidate-proces dat de Ordered List en de Unordered List allebei een color token krijgen.
  - [Avoid large margin for nested lists in rich text](https://github.com/nl-design-system/utrecht/issues/2700)
    Dit is relevant omdat componenten rekening moeten houden met de spacing van componenten ervoor en erna. Hier houden we standaard rekening mee.
  - [Feat/ul style update](https://github.com/nl-design-system/utrecht/pull/3058)
    Deze wijziging verbetert de implementatie van de Unordered List op het gebied van spacing en marker styling. Hoewel de pull request voor de Unordered List is gemaakt, kunnen we de kennis ook meenemen voor de Ordered List.
  - [docs: forced colors story for CSS unordered list](https://github.com/nl-design-system/utrecht/pull/2506)
    Dit is relevant omdat we altijd een forced colors story willen meenemen voor een Candidate component. Hier houden we standaard rekening mee.
- [Den Haag](https://github.com/nl-design-system/denhaag)
  - [Spacing between components](https://github.com/nl-design-system/denhaag/issues/1121)
    Dit is relevant omdat componenten rekening moeten houden met de spacing van componenten ervoor en erna. Hier houden we standaard rekening mee.
- [Uit de GitHub Discussion](https://github.com/orgs/nl-design-system/discussions/230)
  - DSO heeft ook de variant `columns`, waarvoor we een Story meenemen in Test Storybook.
  - Uit Design Open Dag van 30 januari 2026 kwam de vraag naar richtlijnen voor het aantal levels om te ondersteunen en te gebruiken. Om deze vraag te beantwoorden voor Candidate hebben we een onderzoek uitgevoerd.

We hebben besloten om deze issues niet mee te nemen, om de volgende redenen:

- [RVO](https://github.com/nl-design-system/rvo)
  - [Use existing NL Design System community React components where possible](https://github.com/nl-design-system/rvo/issues/265)
    Deze issue is overkoepelend, en het deel relevant voor deze component is al voltooid.
- [Utrecht](https://github.com/nl-design-system/utrecht)
  - [Component: Ordered list](https://github.com/nl-design-system/utrecht/issues/27)
    Deze issue ging om het implementeren van de component. De component is al geïmplementeerd. We hebben de issue gesloten.
  - [feature request: utrecht.ordered-list.marker.color design token](https://github.com/nl-design-system/utrecht/issues/2350)
    Hoewel de vraag relevant is, nemen we deze niet mee in Candidate. Op dit moment wordt dit nog niet toegepast in community-implementaties; het betreft één feature request uit 2024 van een kernteamlid en nog geen bredere behoefte vanuit de community.
  - [alle Angular componenten getest](https://github.com/nl-design-system/utrecht/issues/1589)
    Deze issue gaat om Angular-tests toevoegen in Utrecht.
  - [alle Angular componenten in Storybook](https://github.com/nl-design-system/utrecht/issues/1588)
    Deze issue gaat om Angular-stories toevoegen in Utrecht.
  - [React component library aanbieden](https://github.com/nl-design-system/utrecht/issues/833)
    Deze issue is overkoepelend, en het deel relevant voor deze component is al voltooid.
  - [Meer componenten beschikbaar maken voor Vue 3](https://github.com/nl-design-system/utrecht/issues/1688)
    Deze issue gaat om een Vue-implementatie in Utrecht.
  - [feat: ordered and unordered list item web component](https://github.com/nl-design-system/utrecht/pull/2013)
    Deze issue gaat om een Web Component-implementatie in Utrecht.
- [Den Haag](https://github.com/nl-design-system/denhaag)
  - [Improve and extend Utrecht component design tokens](https://github.com/nl-design-system/denhaag/issues/1081)
    Deze issue is overkoepelend, en het deel relevant voor deze component is al voltooid.
- [Rijkshuisstijl Community](https://github.com/nl-design-system/rijkshuisstijl-community)
  - [Componenten in Storybook met nieuwe Rijkshuisstijl opleveren](https://github.com/nl-design-system/rijkshuisstijl-community/issues/2316)
    Deze issue gaat specifiek om het nieuwe Rijkshuisstijl-design toe te passen in Rijkshuisstijl Community.
  - [Tracking Issue: Twig support voor Drupal MVP](https://github.com/nl-design-system/rijkshuisstijl-community/issues/1307)
    Deze issue gaat om een Twig-implementatie in Rijkshuisstijl Community.
  - [Moved documentation of components Table up to Unordered List into documentation/components folder](https://github.com/nl-design-system/rijkshuisstijl-community/pull/1987)
    Deze issue gaat om het verplaatsen van bestanden binnen de repository.
- [Uit de GitHub Discussion](https://github.com/orgs/nl-design-system/discussions/230)
  - De mogelijkheid om SVG's of andere afbeeldingen als markers te gebruiken nemen we niet mee. Deze kwam alleen terug in oudere ontwerpen van Amsterdam en als idee vanuit DUO, maar wordt op dit moment niet toegepast in de community. Wel hebben we onderzoek gedaan naar de toegankelijkheid van de verschillende implementaties van custom markers.
  - De varianten `group` en `image list` vallen buiten de scope van de Ordered List-component zoals deze binnen NL Design System is gedefinieerd.

Uit [Amsterdam](https://github.com/amsterdam/design-system), [Rotterdam](https://github.com/nl-design-system/rotterdam) en [Tilburg](https://github.com/nl-design-system/tilburg) zijn geen issues bekend gekomen.

Bekijk in [deze Candidate issue](https://github.com/nl-design-system/candidate/issues/1079) hoe we tot dit resultaat zijn gekomen.

### Hoe kun je helpen om deze features wel in de Candidate implementatie te krijgen?

Pas de wijzigingen eerst in de Community implementatie toe en vraag andere organisaties in de community om deze ook in hun projecten te gebruiken. Documenteer jullie bevindingen en successen in de GitHub Discussion, deel deze in de Heartbeat en op Slack. Zo heeft het kernteam meer bewijs dat de feature wenselijk en haalbaar is en kan deze besluiten de functionaliteit alsnog, of in een volgende versie mee te nemen.

**Comment by Rozerinay** [comment]
## Candidate voorbereidingsfase: Versimpeld of gesplitst

De varianten voor de Candidate component zijn bepaald, dat hebben we gedaan op basis van alle voorbeelden die we in deze Discussion hebben gezien. Als een variant veel voor komt, dan is het een algemeen nuttige variant!

### Deze varianten nemen we mee:

#### Item

**Level - enum**

- 1
- 2
- 3

**Marker - enum (design)**

- Default
- Custom

**Marker Text - string (design)**

**Children - HTML (code) / Text - string (design)**

### Mis je een variant?

Laat het in deze Discussion weten als er uit gebruikersonderzoek naar voren is gekomen dat deze design beslissing een hele verbetering zou zijn.

**💡 Tip**
Je kunt [varianten ook als extensie toevoegen](https://www.nldesignsystem.nl/handboek/componenten-vinden/#ik-mis-een-functionaliteit-binnen-een-candidate-of-hall-of-fame-component-wat-nu) aan een component.

**Comment by Rozerinay** [comment]
## Candidate voorbereidingsfase: Anatomie bepaald

<img width="960" height="540" alt="Een geordende lijst met meerdere list items en aanwijzers. De lijst staat binnen een fel-roze gestreept kader. Een aanwijzer met het cijfer 1 wijst naar de volledige geordende lijst. Een aanwijzer met het cijfer 2 wijst naar een afzonderlijk list item met de tekst ‘Text’. Een aanwijzer met het cijfer 3 wijst naar de marker, weergegeven als een volgnummer vóór een list item. Een aanwijzer met het cijfer 4 wijst naar de content van een list item, weergegeven als de tekst ‘Dit is een voorbeeld van een List Item in een Ordered List.’ binnen de lijst" src="https://github.com/user-attachments/assets/5b563585-4dae-408e-a9c2-eda88075e48a" />

De component bestaat uit:

1. Ordered List: de volledige lijst.
2. List Item: een afzonderlijk item binnen de lijst.
3. Marker: voor het weergeven van de volgorde-aanduiding voorafgaand aan de content.
4. Content: voor de inhoud van het List Item.

---

[Deze schets is gemaakt in Figma](https://www.figma.com/design/0Y9Tbb373u6QGxGT6mqUnm/NL-Design-System---Doc-website---Afbeeldingen?node-id=1850-3440)

**Comment by Rozerinay** [comment]
## Candidate voorbereidingsfase: Design tokens bepaald

De design tokens voor de Candidate component zijn bepaald. Dat hebben we gedaan op basis van alle designs die we in deze Discussion hebben gezien. Als een beslissing vaker dan 2 keer voorkomt, is het een algemeen nuttige beslissing.

### Deze design tokens nemen we mee:

**General**
- `nl.ordered-list.color`
- `nl.ordered-list.font-family`
- `nl.ordered-list.font-size`
- `nl.ordered-list.line-height`
- `nl.ordered-list.margin-block-end`
- `nl.ordered-list.margin-block-start`
- `nl.ordered-list.padding-inline-start`

**Item**
- `nl.ordered-list.item.margin-block-end`
- `nl.ordered-list.item.margin-block-start`
- `nl.ordered-list.item.padding-inline-start`

**Marker**
- `nl.ordered-list.marker.color`

### Mis je een design token?

Laat het in deze Discussion weten als er uit gebruikersonderzoek naar voren is gekomen dat deze design-beslissing een hele verbetering zou zijn.

**💡 Tip**
Je kunt design tokens ook als extensie toevoegen aan de component.

- [Voor designers: Zelf componenten uitbreiden](https://www.nldesignsystem.nl/handboek/designer/zelf-componenten-uitbreiden/)
- Voor developers: Op het moment van schrijven is de documentatie hierover nog in ontwikkeling. Vraag ons er gerust naar.
